selfhost/repl.jakt

```diff
@@ -2,10 +2,11 @@ import typechecker { Typechecker, Interpreter, LoadedModule, ModuleId, ScopeId,
 import compiler { Compiler, FileId }
 import lexer { Lexer }
 import parser { Parser }
-import utility { Span, allocate, null }
+import utility { Span }
 import error { JaktError }
 import interpreter { value_to_checked_expression }
-import path { Path }
+import jakt::path { Path }
+import jakt::platform::utility { allocate, null }
```

Comment:
this feels like an implementation detail, not a fan of leaking it like this

Reply:
Yeah, I have changed this back. Maybe in future we need exports or a way to make something private to a module.